docs/conf.py

```diff
@@ -0,0 +1,439 @@
+    "product_page": "launchpad.net",
+    # Product tag image; the orange part of your logo, shown in the page header
+    #
+    # TODO: To add a tag image, uncomment and update as needed.
+    'product_tag': '_static/tag.png',
+    # Your Discourse instance URL
+    #
+    # TODO: Change to your Discourse instance URL or leave empty.
+    #
+    # NOTE: If set, adding ':discourse: 123' to an .rst file
+    #       will add a link to Discourse topic 123 at the bottom of the page.
+    "discourse": "",
+    # Your Mattermost channel URL
+    #
+    # TODO: Change to your Mattermost channel URL or leave empty.
+    # Your Matrix channel URL
+    #
+    # TODO: Change to your Matrix channel URL or leave empty.
+    "matrix": "https://matrix.to/#/#launchpad:ubuntu.com",
+    # Your documentation GitHub repository URL
+    #
+    # TODO: Change to your documentation GitHub repository URL or leave empty.
+    #
+    # NOTE: If set, links for viewing the documentation source files
+    #       and creating GitHub issues are added at the bottom of each page.
+    "github_url": "https://github.com/canonical/launchpad-manual",
+    # Docs branch in the repo; used in links for viewing the source files
+    #
+    # TODO: To customise the branch, uncomment and update as needed.
+    'repo_default_branch': 'main',
+    # Docs location in the repo; used in links for viewing the source files
+    #
+
+
+    # TODO: To customise the directory, uncomment and update as needed.
+    "repo_folder": "/docs/",
+    # TODO: To enable or disable the Previous / Next buttons at the bottom of pages
+    # Valid options: none, prev, next, both
+    "sequential_nav": "none",
+    # TODO: To enable listing contributors on individual pages, set to True
+    "display_contributors": False,
+
+    # Required for feedback button    
+    'github_issues': 'enabled',
+}
+
+# TODO: To enable the edit button on pages, uncomment and change the link to a
+# public repository on GitHub or Launchpad. Any of the following link domains
+# are accepted:
+# - https://github.com/example-org/example"
+# - https://launchpad.net/example
+# - https://git.launchpad.net/example
+#
+html_theme_options = {
+    "sidebar_hide_name": True,
+}
+
+# Project slug; see https://meta.discourse.org/t/what-is-category-slug/87897
+#
+# TODO: If your documentation is hosted on https://docs.ubuntu.com/,
+#       uncomment and update as needed.
+
+slug = ''
+
+#######################
+# Sitemap configuration: https://sphinx-sitemap.readthedocs.io/
+#######################
+
+# Base URL of RTD hosted project
+
+html_baseurl = 'https://canonical-launchpad-manual.readthedocs-hosted.com'
+
+# URL scheme. Add language and version scheme elements.
+# When configured with RTD variables, check for RTD environment so manual runs succeed:
+
+if 'READTHEDOCS_VERSION' in os.environ:
+    version = os.environ["READTHEDOCS_VERSION"]
+    sitemap_url_scheme = '{version}{link}'
```

Comment:
Does Launchpad documentation use this versioning system? Does it need to?
Looks like you only have one set of docs available so you could just use

```suggestion
    sitemap_url_scheme = '{link}'
```

But it's worth talking with the team about if they want to offer versions of the docs, and how you identify changes across versions, etc.

Reply:
We don't offer different versions so I'll commit this suggestion